qiita_pet/handlers/api_proxy/tests/test_studies.py

```diff
@@ -347,8 +347,11 @@ def test_study_files_get_req_multiple(self):
             'remaining': ['uploaded_file.txt'], 'message': '',
             'file_types': [
                 ('raw_forward_seqs', True,
-                 ['test_2.R1.fastq.gz', 'test_1.R1.fastq.gz']),
+                 sorted(['test_2.R1.fastq.gz', 'test_1.R1.fastq.gz'])),
```

Comment:
Would it be better to simply reorder this two element list manually, rather than sort it programmatically?

Reply:
Two additional occurrences below.

Reply:
I mean it's nice to imply that this list needs to be sorted before acting on it, even if/when the contents change.

Reply:
I rather be certain that we always order based on the machine the tests are running on ...

Reply:
ok